.vsts-dotnet-ci.yml

```
@@ -18,36 +32,36 @@ phases:
 
 - template: /build/ci/phase-template.yml
   parameters:
-    name: Windows_x64
-    buildScript: build.cmd
+    name: MacOS
+    buildScript: ./build.sh
     queue:
-      name: Hosted VS2017
+      name: Hosted macOS
 
 - template: /build/ci/phase-template.yml
   parameters:
-    name: core30
+    name: Windows_NetFx
     buildScript: build.cmd
     customMatrixes:
-      Build_Debug_Intrinsics:
-        _configuration: Debug-Intrinsics
-        _config_short: DI
-      Build_Release_Intrinsics:
-        _configuration: Release-Intrinsics
-        _config_short: RI
+      Build_Debug_netfx:
+        _configuration: Debug
```

I just realized this is wrong. This configurations should be have -netfx at the end. You're building for netcoreapp2.1 currently. 
```suggestion
        _configuration: Debug-netfx
```